### ISSUE TITLE
About viewer for Octree-GS

### ISSUE THREAD

**Issue by SmileXUZ** [city-super/Octree-GS#29]

**Comment by SmileXUZ** [comment]
Thanks for your great work, I wanted to use viewer for Octree-GS to observe the reconstructed big scene, but I ran into a problem, first, I used train_bungeenerf.sh to train the data, the data I used was bungeenerf, its data format is shown in the figure:
![程序图片2](https://github.com/city-super/Octree-GS/assets/39118419/3c36092a-21fb-41f4-848a-ea579bbcaeef)
. Then I get the out file, as shown in the figure：
![Uploading 程序图片3.jpg…]()
. In SIBR_viewers, one of the steps is to set Command Arguments: -s Path/to/data -m Path/to/ckpt, but I don't know how to get the ckpt file, can you give me some suggestions?

**Comment by SmileXUZ** [comment]
Thanks for your great work, I wanted to use viewer for Octree-GS to observe the reconstructed big scene, but I ran into a problem, first, I used train_bungeenerf.sh to train the data, the data I used was bungeenerf, its data format is shown in the figure:
![程序图片2](https://github.com/city-super/Octree-GS/assets/39118419/5debb245-2d46-4021-8934-769d8b0f011c)
. Then I get the out file, as shown in the figure：
![程序图片3](https://github.com/city-super/Octree-GS/assets/39118419/98308180-6e4b-459f-bddd-a5040a6c2e6e)
. In SIBR_viewers, one of the steps is to set Command Arguments: -s Path/to/data -m Path/to/ckpt, but I don't know how to get the ckpt file, can you give me some suggestions?

**Comment by tongji-rkr** [comment]
ckpt is the parent directory of `point_cloud`

**Comment by SmileXUZ** [comment]
Thank you very much for your reply. I configured vs2022 according to the tutorial, but there were some errors when I generated sibr_gaussian in Debug mode.
<img width="1270" alt="error" src="https://github.com/city-super/Octree-GS/assets/39118419/755e728a-f6d9-4422-a14d-27126e5e362d">
 I don't know if this is caused by the lack of lib libraries, can you give me some advice?

**Comment by tongji-rkr** [comment]
Check if you use the `Debug` version libtorch:https://drive.google.com/file/d/1ZqcI32kyDSpAdJEVRGmOIl42OepJkEB7/view?usp=sharing

**Comment by tongji-rkr** [comment]
And I can't find `frame` or `_frame` in the code. Did you modify the code?

**Comment by SmileXUZ** [comment]
@tongji-rkr，Thank you for your reply. I did not change the code. This error occurred when I generated the sibr_gaussian project, and it appeared in the pybind11.h file. 
<img width="511" alt="error2" src="https://github.com/city-super/Octree-GS/assets/39118419/b067c723-b484-4ec4-a5ea-5e707aeed75a">
The version of libtorch I use is also downloaded from the tutorial link. I don't know what went wrong. Could it be the wrong python version? I type python in cmd, the version is 3.11.5, and then I type python311.lib in Linker → Input → Additional Dependencies in msvs, and the error still occurs. Even if I change it to python312.lib or python310.lib, I'm confused, I don't know what the problem is.
